webapp/src/app.tsx

```diff
@@ -3853,15 +3853,17 @@ export class ProjectView
         simulator.driver.setSingleSimulator();
     }
 
-    restartSimulator() {
+    restartSimulator(suppressSimulatorFocus?: boolean) {
         const isDebug = this.state.tracing || this.state.debugging;
         if (this.state.simState == SimState.Stopped
             || this.debugOptionsChanged()) {
             this.startSimulator();
         } else {
             simulator.driver.restart(); // fast restart
         }
-        simulator.driver.focus()
+        if (!suppressSimulatorFocus) {
```

Comment:
Does this component already know whether the sim is full screen?
Can we have a method that's shared rather than fetch boadview via ID twice? Are there no refs to boardview?

What about other callers of SimulatorDriver.focus (if any) ? Are they doing the right thing?